staking-pool/README.md

```diff
@@ -298,3 +298,117 @@ pub fn update_reward_fee_fraction(&mut self, reward_fee_fraction: RewardFeeFract
 /// behalf of this staking pool.
 pub fn vote(&mut self, voting_account_id: AccountId, proposal_id: ProposalId) -> Promise;
 ```
+
+## Migrating from an existing validator or contract
+This provides instructions to migrate your staked validator or a validator contract to the new contract for Stakewars2
+
+Upgrade to the latest near-shell:
+```bash
+npm install -g near-shell
+```
+Login:
+```bash
+#If not logged into the browser recover your account with the seed phrase first
+export NEAR_ENV=betanet
```

Comment:
Explain what's this export line. E.g. If running on the `betanet`, switch the near-shell target network to `betanet`